docs/README.md

```diff
@@ -0,0 +1,17 @@
+# Tock documentation
+
+Welcome to the Tock documentation. This documentation is written in
```

Comment:
I think we should consider ditching the "welcome to the tock documentation". I don't think it adds anything, since I already know where I'm at from the header and my position in the filesystem.